pkg/database/migrations/005_receipttableids.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE system_txn_receipts ADD table_ids TEXT;
```

Comment:
Migration that adds the `table_ids` columns. Also, we need to transfer the values in `table_id` to `table_ids`

Reply:
Would it make sense to make `table_ids` "NOT NULL"? Would we always have a `table_id/table_ids` associated with the receipt that we are interested in saving?

Reply:
One issue I see with backfilling is that earlier, we were only saving the last `table_id`. This means that for these rows we will not have the correct value of `table_ids` (all tables associated with the txn). Caveat for future debugging.

Reply:
> Would it make sense to make table_ids "NOT NULL"? Would we always have a table_id/table_ids associated with the receipt that we are interested in saving?

`table_ids` can be null when an error occurs

> One issue I see with backfilling is that earlier, we were only saving the last table_id. This means that for these rows we will not have the correct value of table_ids (all tables associated with the txn). Caveat for future debugging.

That's fine because we shouldn't have any txns with more than one table id in `mainnet`

Reply:
got it, thanks.